package.json

```diff
@@ -41,6 +41,7 @@
     "@babel/core": "^7.20.2",
```

Comment:
![readme](https://user-images.githubusercontent.com/93039359/202204476-21558a84-5216-42cc-8313-ad07b0a8a196.PNG)

**[Optional]**

- [ ] Your test from App.test.js file is not passing 😨 Kindly fix that 👍